apprise/logger.py

```diff
@@ -181,6 +181,7 @@ def __exit__(self, exc_type, exc_value, tb):
         if self.__path:
             # Close our file pointer
             self.__buffer_ptr.close()
+            self.__handler.close()
```

Comment:
Great catch!!

Reply:
The fix is a dedicated commit a566cd838, so I think we would like to keep the commits when merging to master.